src/index.html

```diff
@@ -13,6 +13,36 @@
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__image"></div>
+      <h2 class="card__title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+      <p class="card__code">Product code: 195434</p>
+      <div class="card__rating">
```

Comment:
Add the blank line between the blocks
```suggestion

      <h2 class="card__title">
        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
      </h2>
      
      <p class="card__code">Product code: 195434</p>
      
      <div class="card__rating">
```

Reply:
I have changed this, so we can read code better.